### PR TITLE
chore(storage): Migrate Storage components

### PR DESCRIPTION
**Pull request by ioanabrooks**
#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by thaddmt — APPROVED** [review]

File: packages/react-storage/src/components/StorageImage/_tests_/StorageImage.test.tsx [hunk 1]
```diff
@@ -34,7 +40,7 @@ describe('StorageImage', () => {
         alt="StorageImage"
         className={className}
         imgKey={imgKey}
-        accessLevel="guest"
+        accessLevel={accessLevel}
```
Comment (reesscot) [inline packages/react-storage/src/components/StorageImage/_tests_/StorageImage.test.tsx:43]: Nice, thanks for cleaning this up!

File: packages/react-storage/src/components/StorageManager/ui/FileList/types.ts [hunk 2]
```diff
@@ -10,17 +8,11 @@ export interface FileListProps {
...
-  onPause: (params: {
-    id: string;
-    uploadTask: Storage.UploadDataOutput;
-  }) => void;
-  onResume: (params: {
-    id: string;
-    uploadTask: Storage.UploadDataOutput;
-  }) => void;
+  onPause: (params: { id: string; uploadTask: UploadDataOutput }) => void;
+  onResume: (params: { id: string; uploadTask: UploadDataOutput }) => void;
```
Comment (calebpollman) [inline packages/react-storage/src/components/StorageManager/ui/FileList/types.ts:15]: IMO `uploadTask` name here needs to be reconsidered, and `UploadDataOutput` should be replaced with a scaffold type

**Review by calebpollman — COMMENTED** [review]
LGTM for the most part! Need to audit and remove direct exposing of JS API types from our interfaces, 
and remove the usage of the scoped packages in preference of the umbrella package import paths. There's no guarantee that the scoped package exports will not have breaking changes afaik:
```
import { UploadDataOutput } from '@aws-amplify/storage';
// to
import { UploadDataOutput } from 'aws-amplify/storage';
```

File: packages/react-storage/src/components/StorageManager/types.ts [hunk 3]
```diff
@@ -28,8 +27,7 @@ export interface StorageFile {
   file?: File;
   status: FileStatus;
   progress: number;
-  // uploadTask?: UploadTask;
-  uploadTask?: Storage.UploadDataOutput;
+  uploadTask?: UploadDataOutput;
```
[the remark this answers is not in the document]
Comment (ioanabrooks) [reply packages/react-storage/src/components/StorageManager/types.ts:30]: It is exposed externally, comes into play when overriding components.

**Review by calebpollman — APPROVED** [review]

**Pull request merged**